web/src/settings_config.ts

```diff
@@ -366,6 +366,13 @@ export const waiting_period_threshold_dropdown_values = [
 
 export const retain_message_forever = -1;
 
+export const topic_visibility_policy = {
```

Comment:
We already have
```
export const visibility_policy = {
    INHERIT: 0,
    MUTED: 1,
    UNMUTED: 2,
    FOLLOWED: 3,
};
``` 
in `web/src/user_topics.js`.

Any particular reason for including it here?

Reply:
As other similar objects that store the API values were in settings_config.ts. so I think this should also be here.

Reply:
We don't need a duplicate, `user_topics` is the module that owns this data, so it belongs there, so we can delete this copy.

(`settings_config` is for data that is settings).